### ISSUE TITLE
bug(sbb-map-container): On Safari opened datepickers calendar goes behind the map 

### ISSUE THREAD

**Issue by Acika00mk** [lyne-design-system/lyne-components#2294]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/lyne-design-system/projects/4) for an issue that matches the one I want to file, without success.

### Bug type

Functionality

### It affects the following packages

Components

### Which version of Lyne Components are you using

0.42.6

### Operating system

Any

### Browser / Browser version

Safari

### Input mode

None

### What happened?

When you open the bvi map 
and click on the datepicker icon
the opened calendar is moved behind the map 
It only occurs on Safari

Possible fix is to add 
overflow: -webkit-paged-y;
on the 
sbb-map-container__sidebar 

### Steps to reproduce the issue

1. Open Safari
2. Open BVI Map (https://int-www.sbb.ch/de/fahrplan/bahnverkehrsinformationen/betriebslage-stoerungen.html) 
3. Click on the datepicker 
4. Observe the datepickers calendar is moved behind the map

**Comment by DavideMininni-Fincons** [comment]
Hello @Acika00mk :)
I'm not able to replicate it with the latest version; could you test it again with 0.43.2 ?
Let us know if the problem persists, thanks!

**Comment by federicoisepponfincons** [comment]
I found different ways to solve this, but I cannot say which one is the most convenient to apply / implement

First of all: the problem is directly related to Safari bug that do not show overflowing-x elements when the parent has vertical scrolling, see examples:

![Screenshot 2023-12-22 alle 16 24 08](https://github.com/lyne-design-system/lyne-components/assets/100614752/4b0fb329-f6cd-4856-bc95-f308f5252e64)

![Screenshot 2023-12-22 alle 16 24 24](https://github.com/lyne-design-system/lyne-components/assets/100614752/2c0f0b8b-5c48-483f-a8a8-4ffcb7be4d2b)

1. The proposed solution (add `overflow: -webkit-paged-y;`) is deprecated, it's an old specific no longer used but... it works because it just disables the vertical scrolling, I don't see it usable unless we apply it on the parent, each time we open the datepicker. There are other similar solutions (like apply `display: table;` to the parent), but toggling this to the parent element feels a bit cumbersome.

2. I tried with implementing the `sbb-menu` into the sidebar (with native popover api, see `feat/popover-api` branch), since this element should be attached outside the sidebar itself, therefore it shouldn't be limited overflowing the parent. This solution worked, but only positioning the trigger `sbb-button` inside the sidebar, and the `sbb-menu` outside the `sbb-map-container`

![Screenshot 2023-12-22 alle 16 54 02](https://github.com/lyne-design-system/lyne-components/assets/100614752/8dcf87a0-d525-4a1e-86a7-8396627789e4)

On the other side, this means that the `sbb-datepicker` should be integrated with the native popover, and that could be a bit complicated.

3. Another way could be to place the original `sbb-datepicker` outside the map container, and put inside the sidebar only a trigger that opens it. If this is achievable (and the datepicker remains visible), it shouldn't be affected by this bug the same way as the native popover.
